src/MoBi.Core/Domain/Extensions/ModuleExtensions.cs

```diff
@@ -0,0 +1,12 @@
+using OSPSuite.Core.Domain;
+
+namespace MoBi.Core.Domain.Extensions
+{
+   public static class ModuleExtensions
+   {
+      public static bool IsTemplateMatchFor(this Module module1, Module module2)
```

Comment:
Very unclear what this does. Maybe this should not be an extension but more of a local method. Here we are comparing the name between two modules? Where is this used?